tests/api_samples.py

```diff
@@ -88,3 +206,95 @@
     "reachable": True,
     "last_updated": ["2020-01-31", "15:56:19"],
 }
+PARSED_Z3_SWITCH = {
+    "last_button_event": "short_release",
+    "state": "short_release",
+    "last_updated": ["2019-09-01", "17:45:47"],
+}
+
+# Examples of other kind of sensors in Hue bridge (just for tests)
+MOCK_GEOFENCE = {
```

Comment:
Should This be here?

Reply:
Good catch, it was just for increase test coverage, I'll clean all that. Parsed examples are also not in use now. The _best code is deleted code_ :)